bundles/http/Citrix/SAP-Ariba/Ariba_v2.js

```diff
@@ -361,7 +374,7 @@ function isObject(json) {
 }
 
 function notEmptyArray(json) {
-    return Array.isArray(json) && json.length > 0
+    return Array.isArray(json) && json.length > 0 && json[0] !== undefined
```

Comment:
this extra condition significantly changes the behavior so the name should be changed too. it could be simplified to `return Array.isArray(object) && object[0] !== undefined` also consider cheks for `null`